content/authorization.md

```diff
@@ -1037,9 +1043,11 @@ curl -X POST 'https://authorization.cerner.com/tenants/ec2458f2-1e24-41c8-b71b-0
   -d 'grant_type=client_credentials&scope=system%2FObservation.read%20system%2FPatient.read'
 </pre>
 
-_Note_:  This access model is only supported on resources
-where explicitly indicated in their respective
-FHIR<sup>®</sup> API resource documentation.
+#### JWT Authentication ####
+
+Alternatively, clients may authenticate themselves using signed JSON Web Tokens (JWTs). This process is similar to the above, but uses ``Authorization: Bearer`` instead. This is the preferred mode of authentication for Bulk Data clients. For complete details, see [Java Web Token (JWT)](https://www.rfc-editor.org/rfc/rfc7519) and [JSON Web Token (JWT) Profile for OAuth 2.0 Client Authentication and Authorization Grants](https://www.rfc-editor.org/rfc/rfc7523).
```

Comment:
Note sure if we avoid linking to FHIR spec from this page if not would it be better to make the phrase  'Bulk Data clients' a link that links to this page: http://www.hl7.org/fhir/smart-app-launch/backend-services.html?

Reply:
Sure, added. Better now?

Reply:
yeah, even better they can get to the page I have from the link you have